### PR TITLE
New content

### LINKED ISSUE #1: in progress
Refer to outline in Readme

Deadline: 9/15/2015

### PR DESCRIPTION
**Pull request by SophieDeBenedetto**
needs :eyes: 

closes #1 

@AnnJohn @aviflombaum @deniznida @dfenjves

### CODE REVIEW

**Comment by AnnJohn** [comment]
Assigned to @ipc103 

Deadline- today, 9/15/15

File: README.md [hunk 1]
```diff
@@ -1,13 +1,186 @@
...
+**Important:** Note that we *didn't specify* the "id" column name or value. Since we created the `cats` table with an "id" column whose type is `INTEGER PRIMARY KEY`, we don't have to specify the id column values when we insert data. Primary Key columns are auto-incrementing. As long as you have defined an id column with a data type of `INTEGER PRIMARY KEY`, a newly inserted row's id column will be automatically given the correct value. 
+
+Let's add a few more cats to our table. Create a file, `01_insert_cats_into_cats_table.sql`. Use several `INSERT INTO` statements to insert the following cats into the table:
+
+|id |name| age| breed|
+|---|----|----|------|
+|  1 | "Lil' Bub" | 5 | American Shorthair|
+|  2  | "Hannah" | 1 | Tabby|
+
+Each `INSERT INTO` statement gets it's own line in the `.sql` file in your text editor. Each line needs to end with a `;`.
```
Comment (ipc103) [inline README.md]: Do we need to add instructions for running the .sql file?

**Comment by AnnJohn** [comment]
Also @SophieDeBenedetto how long- ballpark number or range- would it take a student to complete this?

File: README.md [hunk 2]
```diff
@@ -1,13 +1,186 @@
...
+
+Which should give us back:
+
+```bash
+1 | Maru     | 3 | Scottish Fold
+2 | Hannah   | 1 | Tabby
+3 | Lil' Bub | 5 | American Shorthair
+```
+
+But, since we aren't trying to explicitly list the columns we want to select in this case, and are really trying to say "Give me all the data from all the columns for all of the wizards" we can make use of a special selector, known commonly as the "wildcard" selector. You can think of this selector as simply meaning "all" or anything. It is represented by an asterix like `*`. So, to `SELECT` all the data from all of the cats	 we would say:
```
Comment (ipc103) [inline README.md]: I might reword this line as "A faster way to do this is using a special selector, know commonly as the 'wildcard' selector. You can think of this..."

**Comment by ipc103** [comment]
@SophieDeBenedetto made two comments and fixed one typo - let me know what you think - otherwise ready for merge

**Pull request merged**